apps/vr-tests-react-components/src/stories/Card/Card.stories.tsx

```diff
@@ -1,12 +1,14 @@
 import * as React from 'react';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26616&commitId=c4955acb0dbcbfa949fd31854f20d63c28bd21a4&env=prod)

✅ There was 1 screenshots added, 1 screenshots removed, 1842 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged.orientation.chromium.png|0| Added|
|Card Converged.orientation.normal.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv9"/>

Reply:
New tested added here.